code/modules/mob/mob_emote.dm

```diff
@@ -28,6 +28,9 @@
 		return FALSE
 	var/silenced = FALSE
 	for(var/datum/emote/P in key_emotes)
+		// can this mob run the emote at all?
+		if(!P.can_run_emote(src, intentional))
+			continue
```

Comment:
This is already ran as a part of the `if(P.try_run_emote` line below, so I'd rather avoid calling that twice. I think its probably fine to remove the one inside `try_run_emote`, and keep this one. What do you think @lewcc?

Reply:
I agree, looks like you can remove it from there.